Client/sdk/game/CVehicle.h

```diff
@@ -212,6 +212,7 @@ class CVehicle : public virtual CPhysical
     virtual bool           GetTyresDontBurst() = 0;
     virtual unsigned short GetAdjustablePropertyValue() = 0;
     virtual float          GetHeliRotorSpeed() = 0;
+    virtual float          GetPlaneRotorSpeed() const = 0;
```

Comment:
You have added `const` here but not in `CVehicleSA.h` and `CVehicleSA.cpp`. Also, it looks like this method can't be constant?